docs/DeveloperGuide.md

```diff
@@ -284,32 +252,33 @@ class which handles accessing the file as well as converting from JSON into `Top
 
 ![TopicsStorage Class Diagram](./images/TopicsStorage.png)
 
-The format of the JSON file is important as it is loaded in a particular way. This format was designed as an array of topics that hold the different properties for questions, options, hints and explanations. An example is as such:
+The format of the JSON file is important as it is loaded in a particular way. This format has been designed as an array 
+of topics that hold the different properties for questions, options, hints and explanations. An example is as such:
 
 ```json
 [
   {
-    "topic": "OOP",
+    "topic": "Topic Title", 
     "questions": [
       {
-        "description": "What is encapsulation?",
-        "hint": "check the textbook",
-        "explanation": "because option A is the best answer",
-        "options": [
+        "description": "What is your question?",
+        "hint": "Put the hint here",
+        "explanation": "Put the explanation here",
+        "options": [ 
           {
-            "description": "A",
+            "description": "This is the first option and correct answer",
             "correct": true
           },
           {
-            "description": "B",
+            "description": "This is the second option",
             "correct": false
           },
           {
-            "description": "C",
+            "description": "This is the third option",
             "correct": false
           },
           {
-            "description": "D",
+            "description": "This is the fourth option",
```

Comment:
Great explanation given for developers who will use this information to maintain or improve our product!